server/core/src/main/scala/org.apache.daffodil.debugger.dap/Parse.scala

```diff
@@ -157,31 +157,45 @@ object Parse {
   object Debugee {
 
     case class LaunchArgs(
-        schemaPath: Path,
-        dataPath: Path,
+        defaultSchemaPath: Path,
+        defaultDataPath: Path,
         stopOnEntry: Boolean,
         infosetOutput: LaunchArgs.InfosetOutput,
-        tdmlConfig: LaunchArgs.TDMLConfig
+        tdmlConfig: Option[LaunchArgs.TDMLConfig]
     ) extends Arguments {
+      def dataPath: Path =
+        tdmlConfig
+          .map {
+            case LaunchArgs.TDMLConfig(_, name, description, path) =>
+              val (_, dataPath) = TDMLWrapper.execute(defaultSchemaPath, defaultDataPath, name, description, path)
+              dataPath
+          }
+          .getOrElse(defaultDataPath)
+
+      def schemaPath: Path =
```

Comment:
I wasn't sure if `TDMLWrapper.execute` actually did some I/O, so I duplicated the calls for `schemaPath` and `dataPath`. If that should only happen once, we can make a lazy tuple that only calls `TDMLWrapper.execute` once.